s3/src/main/scala/geotrellis/spark/io/s3/S3GeoTiffRDD.scala

```diff
@@ -166,6 +183,48 @@ object S3GeoTiffRDD {
     *
     * @param bucket Name of the bucket on S3 where the files are kept.
     * @param prefix Prefix of all of the keys on S3 that are to be read in.
+    * @param keyTransform function to transform input key basing on the URI information.
+    */
+  def singleband[I, K](bucket: String, prefix: String, keyTransform: (URI, I) => K, options: Options)(implicit sc: SparkContext, rr: RasterReader[Options, (I, Tile)]): RDD[(K, Tile)] =
+    apply[I, K, Tile](bucket, prefix, keyTransform, options)
```

Comment:
`keyTransform` is a little overloaded with the way we use it for SFC. This should be renamed to `uriToKey` or something similar.